src/Settings.php

```diff
@@ -139,7 +139,7 @@ public static function parseArguments(array $arguments)
            -m \
            --first-parent \
            --format=format: \
-           -- "${PATH_ARGS[@]}" | egrep -v '^$' || :";
+           -- "'*' . array('.php', '.phtml', '.php3', '.php4', '.php5')" | egrep -v '^$' || :";
```

Comment:
What do this line? It do concat string with array. I mean that a result of this expression have non-sence.